app/src/main/res/values-cn/strings.xml

```diff
@@ -0,0 +1,135 @@
+<resources>
+    <string name="app_name" translatable="false">Vigilante</string>
```

Comment:
you won't be needing the app name, translatable is false, you can delete it

Reply:
@FunkyMuse I added <string name="chinese">translation</string>. Could I keep the line of the app name? I'd like to keep the line number of strings,xml for Chinese same as raw strings.xml for easier maintenance.

Reply:
It's not good to keep it for future references, it doesn't add any value, i get your point but everywhere else is removed so...

Reply:
@FunkyMuse Removed.